practice-two/src/components/FormValidation/FormValidation.jsx

```diff
@@ -121,12 +111,25 @@ const FormValidation = ({
       setErrors(newErrors);
 
       if (Object.keys(newErrors).length === 0) {
```

Comment:
Do you need to check error here when already to check error on blur function?

Reply:
Yes, I do
Because onBlur function only check when user click outside after clicking inside input.
To solve this problem when user forgets typing in some field and click submit button, I will check form again in submit by using loop to loop through each input field one more to check
If having error, it will store a error into `newError` object.
After that, I will get all the keys in `newError` object to change them to array by the way using Object.key and using this array to check